javascript/general_style_guide.md

```diff
@@ -15,6 +15,18 @@ Please use always triple equal comparisions and get rid of a lot of bugs related
 "0" === 0 // false
 ```
 
+### Strings
+
+Use single quotes for strings: `'This is a regular string` 
+
+Use template strings for interpolation or multiline strings:
```

Comment:
Is "template strings" a term?

How about "backticks and template strings", to clarify the quoting?

Reply:
Explanation in the template string example.